scripts/utils.py

```diff
@@ -390,7 +390,7 @@ def transform(
          !! Missing template support for 'conj' (word is 'foo')
         ''
     """
-
+    debug = True
```

Comment:
I would vote to always show the message, and remove the "debug" stuff completely. It will force us to move forward.

Reply:
Sorry... it was not meant to be commited... time to go to bed...

Reply:
Me too haha